lightning/src/ln/channelmanager.rs

```diff
@@ -6324,7 +6361,9 @@ where
 		}
 	}
 
-	fn claim_funds_from_hop<ComplFunc: FnOnce(Option<u64>, bool) -> Option<MonitorUpdateCompletionAction>>(
+	fn claim_funds_from_hop<
+		ComplFunc: FnOnce(Option<u64>, bool) -> (Option<MonitorUpdateCompletionAction>, Option<RAAMonitorUpdateBlockingAction>)
```

Comment:
Looks like `(None, Some)` is not possible. Should we consider a different nesting or possibly a three-variant enum?